store.go

```diff
@@ -362,3 +370,16 @@ func (c *controller) processEndpointUpdate(ep *endpoint) bool {
 
 	return false
 }
+
+func ensureKeys(key string, cs datastore.DataStore) error {
+	exists, err := cs.KVStore().Exists(key)
+	if err != nil {
+		return err
+	}
+	if !exists {
```

Comment:
Can we refactor this snippet as follows to avoid multiple indentation?
```
if exists {
    return nil
}

return cs.KVStore().Put(key, []byte{}, nil)
```